ui.frontend/html/src/components/AddContactView/index.jsx

```diff
@@ -5,7 +5,7 @@ import React from 'react'
 
 
 export default
-@inject('mainUI')
+@inject('core')
```

Comment:
What causes the need for this change? I can't find a matching change that renaming something from 'mainUI' to 'core'. Or am I missing something? (I have no idea what the `@inject()` call does/is used for.)

Reply:
The previous version of the AddContactView was integrated in the main view. Therefore the mainUI store was used to trigger the change of the view. Now, the core store is needed in order to call the corresponding java method.

In an upcoming PR I will rework the whole store setup.